openapi/src/main/java/io/helidon/openapi/OpenApiFeature.java

```diff
@@ -68,6 +68,15 @@ public abstract class OpenApiFeature extends HelidonFeatureSupport {
      */
     public static final String DEFAULT_CONTEXT = "/openapi";
 
+    /**
+     * Returns a new builder for preparing an SE variant of {@code OpenApiFeature}.
+     *
+     * @return new builder
+     */
+    public static Builder<?, ?> builder() {
```

Comment:
Returning `Builder<?, ?>` does not seem right. This is awkward when assigned to a variable. Is the abstraction really needed here? 
If we switch to new builder API, this should return the OpenApiConfig.Builder (and the MpOpenApiConfig would extend `OpenApiConfig.BuilderBase<?, ?>`).

Reply:
We can keep this until we refactor to new builder API

Reply:
For the sake of speed I will leave this as-is for now.